.github/workflows/publish-package.yml

```diff
@@ -20,27 +20,34 @@ jobs:
     name: Build and publish Python distro to PyPI
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1.2.0
+    - uses: actions/checkout@v1
+      if: github.event_name == 'workflow_dispatch'
       with:
-        ref: ${{ github.event.inputs.tag }}"
+        ref: ${{ github.event.inputs.tag }}
+    - uses: actions/checkout@v2
+      if: github.event_name != 'workflow_dispatch'
 
     - name: Set up Python 3.7
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
       with:
         python-version: 3.7
-    - name: Install pep517
+
+    - name: Install build
       run: |
-        python -m pip install pep517 --user
+        python -m pip install build
```

Comment:
pep517 is not intended to be a user-facing tool. See https://github.com/pypa/pep517/issues/91